resources/views/dashboard/training/new_ticket.blade.php

```diff
@@ -111,13 +111,13 @@
             <div class="col-sm-6">
                 <div class="form-group">
                     <label for="comments" class="form-label">Comments (Visible to Controller and other Trainers)</label>
-                    {{ html()->textarea('comments', null)->class(['form-control']) }}
+                    <textarea name="comments" class="form-control text-editor" id="comments"></textarea>
                 </div>
             </div>
             <div class="col-sm-6">
                 <div class="form-group">
                     <label for="trainer_comments" class="form-label">Trainer Comments (Visible to Only Other Trainers)</label>
-                    {{ html()->textarea('trainer_comments', null)->class(['form-control']) }}
+                    <textarea name="trainer_comments" class="form-control text-editor" id="trainer_comments"></textarea>
```

Comment:
Let's do this the same everywhere. In `.../edit_ticket.blade.php` it looks like we're using `html()->` method, and then here it's swapped to `<textarea ...></textarea>`. I'd prefer to see this done everywhere with the `html()->` for consistency